src/test/regress/multi_schedule

```diff
@@ -66,7 +66,7 @@ test: multi_basic_queries multi_complex_expressions multi_subquery multi_subquer
 test: multi_subquery_complex_reference_clause multi_subquery_window_functions multi_view multi_sql_function multi_prepare_sql
 test: sql_procedure multi_function_in_join
 test: multi_subquery_in_where_reference_clause
-test: multi_subquery_union multi_subquery_in_where_clause multi_subquery_misc
+test: multi_subquery_union multi_subquery_in_where_clause multi_subquery_misc full_join
```

Comment:
Very minor: Could you maybe add the test above line? This line has 3, the above is only 1 currently.